scipy/constants/__init__.py

```diff
@@ -5,316 +5,107 @@
 
 .. currentmodule:: scipy.constants
 
-Physical and mathematical constants and units.
 
+About the SI units
+==================
+In 2019 seven système international d’unités (SI) base units were (re)defined to
+serve as a basis for all physical quantities by the bureau
+international des poids et mesures (BIPM) [Introduction]_. They can be summarized as followed
+even though for a detailed background the given link is recommended:
 
-Mathematical constants
-======================
+- The second :math:`(\mathrm{s})` as the SI unit of time is derived from the
+  unperturbed ground-state hyperfine transition frequency of the caesium-133
+  atom :math:`\Delta \nu_{\mathrm{Cs}}`.
 
-================  =================================================================
-``pi``            Pi
-``golden``        Golden ratio
-``golden_ratio``  Golden ratio
-================  =================================================================
+- The metre :math:`(\mathrm{m})` as the SI unit of length is derived from the
+  speed of light in vaccum :math:`c`.
 
+- The kilogram :math:`(\mathrm{kg})` as the SI unit of mass is derived from the
+  Planck constant :math:`h`.
 
-Physical constants
-==================
+- The ampere :math:`(\mathrm{A})` as the SI unit of electric current is derived
+  from the elementary charge :math:`e`.
+
+- The kelvin :math:`(\mathrm{K})` as the SI unit of thermodynamic temperature is
+  derived from the Boltzmann constant :math:`k`.
+
+- The mole :math:`(\mathrm{mol})` as the SI unit of amount of substance is
+  derived from the Avogadro constant :math:`N_{\mathrm{A}}`.
+
+- The candela :math:`(\mathrm{cd})` as the SI unit of luminous intensity in a
+  given direction is derived from the luminous efficacy of monochromatic
+  radiation :math:`K_{\mathrm{cd}}`.
+
+All physical quantities and "traditional" units such as newton or pascal
+can expressed by a combination of these base units. Therefore the BIPM
+lists in addition to the seven base units 22 derived units with special
+names and serveral non-SI units. Of course all of before mentioned units
+can be used with the SI prefixes.
+
+However, one has to keep in mind that different systems of measurements apply
+for different problems and therefore the SI units are not necessarily the
+best (in terms of convenience) units for every problem. An overview of
+different systems of measurement can be found on Wikipedia [Wikipedia]_ .
 
-===========================  =================================================================
-``c``                        speed of light in vacuum
```

Comment:
Actually, this you removed and didn't put back. Same for the other ones - that explains the CI failure